apps/soft_pwm_test/pkg.yml

```diff
@@ -26,9 +26,6 @@ pkg.keywords:
 
 pkg.deps:
     - kernel/os
-   # - hw/hal
-   # - hw/cmsis-core
     - sys/console/full
-   # - hw/bsp/nrf52dk
```

Comment:
Instead of commenting lines out as obsolete, it's better to remove them. If you decide later that you want to bring those changes back, they'll be visible via the file history.

Reply:
These comments are the kind of thing I usually address when cleaning up.
I am actually considering to delete this app. We may simply have pwm_test using any pwm driver, by just enabling whatever driver the user wants with newt setting a given syscfg variable.

Reply:
That sounds good to me.